crates/kubelet/src/node.rs

```diff
@@ -389,3 +383,66 @@ fn lease_spec_definition(node_name: &str) -> serde_json::Value {
         }
     )
 }
+
+/// Defines the labels that will be applied to this node
+///
+/// Default values and passed node-labels arguments are injected by config.
+fn node_labels_definition(arch: &str, config: &Config) -> HashMap<String, String> {
+    // Add mandatory static labels
+    let mut labels = HashMap::new();
+    labels.insert("beta.kubernetes.io/os".to_string(), "linux".to_string());
```

Comment:
can you switch these to a `to_owned`? While both achieve the same result, one is slightly less computationally expensive since we don't have to do any type casting from an `&str` to a string (at least, that's my understanding).

Reply:
http://www.lowlevelmanager.com/2016/02/rust-tostring-vs-toowned-for-string.html

Reply:
Done.